automation/jleechanorg_pr_automation/openai_automation/codex_github_mentions.py

```
@@ -0,0 +1,563 @@
+    )
+
+    parser.add_argument(
+        "--debug",
+        action="store_true",
+        help="Debug mode: take screenshots, save HTML, keep browser open"
+    )
+
+    args = parser.parse_args()
+
+    if args.verbose:
+        logger.setLevel(logging.DEBUG)
```

Ruff (0.14.8)

Undefined name `logging`

(F821)